pkg/config/analysis/local/istiod_analyze.go

```diff
@@ -376,28 +377,30 @@ func (sa *IstiodAnalyzer) AddRunningKubeSourceWithRevision(c kubelib.Client, rev
 	// TODO: are either of these string constants intended to vary?
 	// We gets Istio CRD resources with a specific revision.
 	krs := sa.kubeResources.Remove(kuberesource.DefaultExcludedSchemas().All()...)
-	if remote {
-		krs = krs.Remove(kuberesource.DefaultRemoteClusterExcludedSchemas().All()...)
-	}
-	store := crdclient.NewForSchemas(c, crdclient.Option{
-		Revision:     revision,
-		DomainSuffix: "cluster.local",
-		Identifier:   "analysis-controller",
-		FiltersByGVK: map[config.GroupVersionKind]kubetypes.Filter{
-			gvk.ConfigMap: {
-				Namespace:    sa.istioNamespace.String(),
-				ObjectFilter: kubetypes.NewStaticObjectFilter(isIstioConfigMap),
+
+	if !remote {
+		store := crdclient.NewForSchemas(c, crdclient.Option{
+			Revision:     revision,
+			DomainSuffix: "cluster.local",
+			Identifier:   "analysis-controller",
+			FiltersByGVK: map[config.GroupVersionKind]kubetypes.Filter{
+				gvk.ConfigMap: {
+					Namespace:    sa.istioNamespace.String(),
+					ObjectFilter: kubetypes.NewStaticObjectFilter(isIstioConfigMap),
+				},
 			},
-		},
-	}, krs)
-	sa.stores = append(sa.stores, store)
+		}, krs)
+		sa.stores = append(sa.stores, store)
+	}
 
 	// We gets service discovery resources without a specific revision.
 	krs = sa.kubeResources.Intersect(kuberesource.DefaultExcludedSchemas())
 	if remote {
-		krs = krs.Remove(kuberesource.DefaultRemoteClusterExcludedSchemas().All()...)
+		for _, multiAnalyzer := range multicluster.AllMultiCluster() {
+			krs = kuberesource.ConvertInputsToSchemas(multiAnalyzer.Metadata().Inputs)
```

Comment:
This will override the previous one if we have more analyzers. Should be a combination of the schemas.

Reply:
yep, let me fix it